doc/src/vpr/command_line_usage.rst

```diff
@@ -1730,6 +1730,16 @@ Analysis Options
 
     **Default:** ``unconnected``
 
+.. option:: --post_synth_netlist_module_parameters { on | off }
+
+    Controls whether the post-synthesis netlist modules should use Verilog parameters
```

Comment:
Maybe make clear this is output, e.g. "... any post-synthesis netlist output by VTR can use Verilog parameters in its modules."